.github/workflows/test-linux.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332  # v4.1.7
         with:
           persist-credentials: false
+          path: node
```

Comment:
Is there a reason why node isn't checked out directly into the weird path characters?

Reply:
Yes, https://github.com/nodejs/node/pull/48409#issuecomment-1585705604